test/common/upstream/health_checker_impl_test.cc

```diff
@@ -129,6 +129,21 @@ class HttpHealthCheckerImplTest : public testing::Test {
   HttpHealthCheckerImplTest()
       : cluster_(new NiceMock<MockClusterMockPrioritySet>()),
         event_logger_(new MockHealthCheckEventLogger()) {}
+  template <typename Config> void http_health_check_helper(Config conf, bool addHostCheck = true) {
```

Comment:
Can you read through https://github.com/envoyproxy/envoy/blob/master/STYLE.md for basic Envoy naming conventions, etc? In particular, with the naming convention for class methods, you would need to call this `httpHealthCheckHelper(Config conf, bool add_host_check = true)`. ie your naming style for the function and its second param are reversed  :)